src/Transition/CreateNewIDFUsingRulesV9_5_0.f90

```diff
@@ -435,6 +439,101 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 
               ! If your original object starts with Z, insert the rules here
 
+              CASE('ZONEHVAC:LOWTEMPERATURERADIANT:VARIABLEFLOW')
+                CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                OutArgs(1)=InArgs(1)
+                OutArgs(2)=TRIM(InArgs(1))//(' Design Object')
+                OutArgs(3:5)=InArgs(2:4)
+                OutArgs(6)=InArgs(8)
+                OutArgs(7)=InArgs(13)
+                OutArgs(8:10)=InArgs(16:18)
+                OutArgs(11)=InArgs(22)
+                OutArgs(12:14)=InArgs(25:27)
+                OutArgs(15:16)=InArgs(32:33)
+                CurArgs = 16
+
+                CALL GetNewObjectDefInIDD('ZoneHVAC:LowTemperatureRadiant:VariableFlow:Design',NumArgs,AorN,ReqFld,ObjMinFlds,FldNames,FldDefaults,FldUnits)
+                POutArgs(1) = OutArgs(2)
+                POutArgs(2:4) = InArgs(5:7)
+                POutArgs(5:8) = InArgs(9:12)
+                POutArgs(9:10) = InArgs(14:15)
+                POutArgs(11:13) = InArgs(19:21)
+                POutArgs(14:15) = InArgs(23:24)
+                POutArgs(16:19) = InArgs(28:31)
+                IF (InArgs(34) /= Blank) THEN
+                  CurVarIterator = 20
+                  POutArgs(CurVarIterator) = InArgs(34)
+                END IF
+                IF (InArgs(34) == Blank) THEN
+                  CurVarIterator = 19
+                END IF
```

Comment:
Added logic for the last line in the Design objects.

Reply:
next time:

```
CurVarIterator = 19
IF (InArgs(34) /= Blank) THEN
  CurVarIterator = 20
  POutArgs(CurVarIterator) = InArgs(34)
END IF
```